things/elevator/modbus/js/main.js

```diff
@@ -6,8 +6,8 @@ const { parseArgs } = require('node:util')
 require('dotenv').config()
 
 const thingName = "modbus-elevator"
-const hostname = "0.0.0.0"
-let portNumber = "8502"
+const hostname = process.env.HOSTNAME ?? "0.0.0.0"
```

Comment:
localhost doesn't work?

Reply:
It doesn't work, I assume it is because how it is implemented in https://www.npmjs.com/package/modbus-serial.

Reply:
We can do such that if the environment variable has `localhost`, we change it in this code to `0.0.0.0`